src/v1/Groth16VerifierExtension.sol

```diff
@@ -265,8 +264,7 @@ contract Groth16VerifierExtension is Verifier {
         virtual
     {
         require(
-            isCDK() || blockHash == expectedBlockHash,
-            "Block hash must equal as expected."
+            blockHash == expectedBlockHash, "Block hash must equal as expected."
```

Comment:
note, this function is overwritten in the QueryManager. We can avoid the confusion overwriting once [this is addressed](https://linear.app/lagrange/issue/LAG-192/consolidate-contracts-in-lagrange-lpn-contracts-repo).